Gemfile.lock

```diff
@@ -0,0 +1,82 @@
+PATH
```

Comment:
Gemfile.lock is listed in the gitignore. How did you even check this file in? Regardless, it shouldn't be here.

Reply:
I believe it might be added later and the PR was a bit older so it already had it in there; i removed it now.